tests/components/onewire/test_entity_sysbus.py

```diff
@@ -1,129 +1,121 @@
-        assert registry_entry is not None
-        assert (
-            registry_entry.unique_id == f"/sys/bus/w1/devices/{MOCK_DEVICE_ID}/w1_slave"
-        )
-        assert registry_entry.unit_of_measurement == TEMP_CELSIUS
-
-        # 25.123
-        current_time = datetime.now()
-        state = hass.states.get(MOCK_ENTITY_ID)
-        assert state.state == "25.1"
-
-        # FileNotFoundError
-        current_time = current_time + timedelta(minutes=2)
-        async_fire_time_changed(hass, current_time)
-        await hass.async_block_till_done()
-        state = hass.states.get(MOCK_ENTITY_ID)
-        assert state.state == "unknown"
-
-        # 25.223
-        current_time = current_time + timedelta(minutes=2)
-        async_fire_time_changed(hass, current_time)
-        await hass.async_block_till_done()
-        state = hass.states.get(MOCK_ENTITY_ID)
-        assert state.state == "25.2"
-
-        # InvalidCRCException
-        current_time = current_time + timedelta(minutes=2)
-        async_fire_time_changed(hass, current_time)
+        assert await async_setup_component(hass, SENSOR_DOMAIN, MOCK_CONFIG)
         await hass.async_block_till_done()
-        state = hass.states.get(MOCK_ENTITY_ID)
-        assert state.state == "unknown"
 
-        # 25.323
-        current_time = current_time + timedelta(minutes=2)
-        async_fire_time_changed(hass, current_time)
-        await hass.async_block_till_done()
-        state = hass.states.get(MOCK_ENTITY_ID)
-        assert state.state == "25.3"
-
-        # NotFoundSensorException
-        current_time = current_time + timedelta(minutes=2)
-        async_fire_time_changed(hass, current_time)
-        await hass.async_block_till_done()
-        state = hass.states.get(MOCK_ENTITY_ID)
-        assert state.state == "unknown"
-
-        # 25.423
-        current_time = current_time + timedelta(minutes=2)
-        async_fire_time_changed(hass, current_time)
-        await hass.async_block_till_done()
-        state = hass.states.get(MOCK_ENTITY_ID)
-        assert state.state == "25.4"
+    assert len(entity_registry.entities) == len(expected_sensors)
 
-        # UnsupportResponseException
-        current_time = current_time + timedelta(minutes=2)
-        async_fire_time_changed(hass, current_time)
-        await hass.async_block_till_done()
-        state = hass.states.get(MOCK_ENTITY_ID)
-        assert state.state == "unknown"
-
-        # 25.523
-        current_time = current_time + timedelta(minutes=2)
-        async_fire_time_changed(hass, current_time)
-        await hass.async_block_till_done()
-        state = hass.states.get(MOCK_ENTITY_ID)
-        assert state.state == "25.5"
+    for expected_sensor in expected_sensors:
+        entity_id = expected_sensor["entity_id"]
+        registry_entry = entity_registry.entities.get(entity_id)
+        assert registry_entry is not None
+        assert registry_entry.unique_id == expected_sensor["unique_id"]
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6333224)